src/bin/contribute.rs

```diff
@@ -985,6 +987,32 @@ impl Contribute {
             self.notify_contribution(&chunk_id, serde_json::to_value(signed_data)?)
                 .await?;
 
+            if chunk_id == "0" {
+                // load data from attestation
+                match fs::read_to_string("plumo.attestation") {
+                    Ok(line) => {
+                        let arr: Vec<&str> = line.split(" ").collect();
+                        if arr.len() == 3 {
+                            let data = json!({
+                                "id": arr[0],
+                                "address": arr[1],
+                                "data": arr[2],
+                            });
+
+                            let signed_data = SignedData {
+                                signature: sign_json(&self.private_key, &data)?,
+                                data: data,
+                            };
+                            self.add_attestation(serde_json::to_value(signed_data)?)
+                                .await?;
+                        } else {
+                            warn!("plumo.attestation had bad format");
+                        }
+                    }
+                    Err(err) => warn!("Couldn't read plumo.attestation {}", err),
```

Comment:
Can we do something more interesting than warning? Maybe reporting an error to the server? Not sure if worth it.